ethpm_cli/parser.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+from pathlib import Path
+
+
+def get_ethpm_parser() -> argparse.ArgumentParser:
```

Comment:
This is fine, though I think it's also fine to just define the parser at the top level of the module and import it directly rather than rebuilding it each time.